config/locales/views/about/en.yml

```diff
@@ -18,3 +18,13 @@ en:
     contributors_heading: "CONTRIBUTORS"
     contributors_description: "Meet the awesome people of CircuitVerse community that've made this platform what it is now."
     become_contributor: "Become a Contributor"
+    mentorship_programs_heading: "MENTORSHIP PROGRAMS" 
+    mentorship_programs_description: "CircuitVerse has been participating in various mentoring programs, including:"
+    program_gsoc: "Google Summer of Code "
+    program_gsoc_timeline: "2020, 2021, 2022, 2023"
```

Comment:
Make the years embed with respective program circuitverse page, for example : [2021](https://summerofcode.withgoogle.com/archive/2021/organizations/6293580166987776)

Note: you can make translation act as html by suffixing the key by `_html` for example `program_gsoc_timeline_html`